sql/testdata/datetime

```diff
@@ -264,3 +264,114 @@ SELECT extract(nanosecond from timestamp '2001-04-10 12:04:59.34565423')
 
 query error extract: unsupported timespan: nansecond
 SELECT extract(nansecond from timestamp '2001-04-10 12:04:59.34565423')
+
+# Test SET TIME ZONE
+
+# default time zone of UTC
+query TTT
+SELECT '2015-08-25 05:45:45.53453'::timestamp
+----
+2015-08-25 05:45:45.53453 +0000 UTC
+
+statement ok
+SET TIME ZONE 'Europe/Rome'
+
+query TTT
+SELECT '2015-08-25 05:45:45.53453 CET'::timestamp
+----
+2015-08-25 04:45:45.53453 +0000 UTC
+
+statement ok
+SET TIME ZONE "America/New_York"
+
+query TTT
+SELECT '2015-08-24 23:45:45.53453 EST'::timestamp
+----
+2015-08-25 04:45:45.53453 +0000 UTC
+
+query TTT
+SELECT '2015-08-24 23:45:45.53453 EDT'::timestamp
+----
+2015-08-25 03:45:45.53453 +0000 UTC
+
+# PST is not interpretted properly.
```

Comment:
It's in a different time zone. You have to set the correct time zone to have the data interpreted correctly, and unfortunately there are no errors or warning returned.